src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -283,32 +283,36 @@ protected override void CompleteAsyncMethodChecksBetweenStartAndFinish()
             }
         }
 
-        public override ImmutableArray<TypeParameterConstraintClause> GetTypeParameterConstraintClauses(bool canIgnoreNullableContext)
+        public override ImmutableArray<TypeParameterConstraintClause> GetTypeParameterConstraintClauses(bool canUseLightweightTypeConstraintBinding)
         {
-            if (!_lazyTypeParameterConstraints.HasValue(canIgnoreNullableContext))
+            var clauses = _lazyTypeParameterConstraints;
```

Comment:
>clauses = _lazyTypeParameterConstraints [](start = 16, length = 39)

Please consider inlining this local to avoid risk of using the (stale) local later in the method.